package.json

```diff
@@ -26,14 +26,13 @@
     "@angular/common": ">=4.3.6",
     "@angular/core": ">=4.3.6",
     "@skyux/core": "^3.5.1",
-    "@skyux/i18n": "^3.5.0",
-    "@skyux/theme": "^3.4.0"
```

Comment:
Maybe a dumb question - won't this break all the mixins imports if we remove theme?

Reply:
Not dumb at all. Removing the peer won't have any effect on the SCSS files because those are compiled into normal CSS and injected into the newly created JavaScript file for each component. The only time we need a peer is when the TypeScript file is importing from a third-party library, because the contents of the peer aren't bundled with the resulting JavaScript file.

Reply:
Makes sense. So before this compiles into CSS, I'm assuming those import paths in the SCSS files are still valid because we have a dev dependency of `@blackbaud/skyux` ?

Reply:
Correct!